autoemulate/experimental/exploratory/active_matter/README.md

```diff
@@ -0,0 +1,18 @@
+Notebook requirements:
```

Comment:
Maybe add a little note to the README that this is about exploration of one of the Well datasets?